src/MessageTrait.php

```diff
@@ -3,6 +3,8 @@
 
 use Psr\Http\Message\StreamInterface;
 
+require_once __DIR__.'/functions.php';
```

Comment:
Agreed. After some debug I found it is possible issue with composer, which does not include file in case of forked project. Reported: https://github.com/composer/composer/issues/5451 Will see if I missed something or not.